clientd3d/statgame.c

```diff
@@ -370,13 +381,31 @@ void GameTimer(HWND hwnd, UINT id)
 /****************************************************************************/
 void GameIdle(void)
 {
+   static const std::chrono::duration<double> minInterval(1.0 / 60.0); // 1/60th of a second
+   static auto lastCallTime = std::chrono::high_resolution_clock::now();
```

Comment:
This is not the last call time, it's the last time the function was allowed to run to completion.